common/src/main/resources/assets/wynntils/urls.json

```diff
@@ -16,6 +16,14 @@
       "version"
     ]
   },
+  {
```

Comment:
Reminder to self: Also post this on api website.

Reply:
https://github.com/Wynntils/WynntilsWebsite-API/pull/47

I will push that at the same time that this goes in (it's not that important to sync them, but it seems reasonable to push them roughly at the same time)